SFX-96 Setup TypeDoc for View Layer (#2)
groupby/sfx-view: README.md

yarn build
```

> 

Author: That is where the suggestion applied and as you can see in the latest version of readme it is all fixed.

The following command will generate documentation for each module in the `packages` directory. It uses [TypeDoc](https://typedoc.org/) and outputs to the `docs` directory.

```sh
yarn dev
```
##Documentation

> The following command will generate documentation in the `docs` directory for each module in the `packages` directory using [TypeDoc](https://typedoc.org/).

Reviewer: Use @DanielMoniz-GBI's suggestion here instead.

Author: that is his suggestion... https://groupbyinc.slack.com/archives/CHUMB6UM6/p1561401593150300?thread_ts=1561399180.149300&cid=CHUMB6UM6

Author: @jmbtutor the suggestion applied to the wrong line. I applied the suggestion to the correct line, and fixed the wrong line it had overridden. If you look at the latest version of readme it should all be fixed :)

```sh
yarn dev
```
##Documentation

> The following command will generate documentation for each module in packages/sfx using typeDoc. This will update the index file in the docs folder.

Reviewer: ```suggestion
The following command will generate documentation in the `docs` directory for each module in the `packages` directory using [TypeDoc](https://typedoc.org/).
```

Mentioning TypeDoc may not be necessary.

Reviewer 2: ```suggestion
The following command will generate documentation for each module in the `packages` directory. It uses [TypeDoc](https://typedoc.org/) and outputs to the `docs` directory.
```

Reviewer 2: (the latter one is a bit less confusing because it's just in two sentences)
